### PR TITLE
✨ Get instances and tasks by identity

### PR DESCRIPTION
**Pull request by ElRaptorus**
**Changes:**

1. Add `flowNodeInstanceId` to all `BpmnMessages`
2. Remove unused `ProcessEndedMessage` and `ProcessTerminatedMessage`
    - These types of notifications use the `EndEventReached` and `TerminateEndEventReached` message types, located under the `BpmnMessage` namespace
3. Refactor `EventAggregatorSettings`:
    - Merge `routePaths` and `messagePaths` into `messagePaths`.
    - Rename `routeParams` to `messageParams`.
    - Split messages into `internal` and `public` message types to differentiate between messages passed inside the process engine and messages that get send to a public subscriber.
    - Public messages will not contain the identity of the ProcessInstanceOwner, but the internal messages will.

**Issues:**

Part of https://github.com/process-engine/process_engine_runtime/issues/198

PR: #36

## How can others test the changes?

Use the refactored messages with the EventAggregator.

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.).
- [x] I've rebased the `develop` branch with my branch before finishing this PR.
- [x] I've **summarized all changes** in a list above.
- [x] I've mentioned all **PRs, which relate to this one**.
- [x] I've prefixed my Pull Request title is according to [gitmoji guide](https://gitmoji.carloscuesta.me/).

### CODE REVIEW

File: src/data_models/messages/event_aggregator_settings.ts [hunk 1]
```diff
@@ -1,14 +1,5 @@
...
-  userTaskReached: 'user_task_reached',
-  userTaskFinished: 'user_task_finished',
-  manualTaskReached: 'manual_task_reached',
-  manualTaskFinished: 'manual_task_finished',
-  processTerminated: 'process_terminated',
-  processStarted: 'process_started',
-  processEnded: 'process_ended',
-};
-
+// tslint:disable:max-line-length
```
Comment (Vyperus) [inline src/data_models/messages/event_aggregator_settings.ts]: Is that needed here?

File: src/data_models/manual_task/manual_task.ts [hunk 2]
```diff
@@ -9,4 +11,5 @@ export class ManualTask {
   public processModelId: string;
   public processInstanceId?: string;
   public tokenPayload: any;
+  public owner: IIdentity;
```
Comment (roblen45) [inline src/data_models/manual_task/manual_task.ts]: processInstanceOwner

File: src/data_models/user_task/user_task.ts [hunk 3]
```diff
@@ -19,4 +21,5 @@ export class UserTask {
    * The token payload the UserTask got suspended with.
    */
   public tokenPayload: any;
+  public owner: IIdentity;
```
Comment (roblen45) [inline src/data_models/user_task/user_task.ts]: processInstanceOwner

**Comment by ElRaptorus** [comment]
@roblen45 done

**Pull request merged**